### PR TITLE
Added auto pin and auto timeout

### LINKED ISSUE #3: Autopin
Klasika - [nastavitelne mnozsti] 📌 => pin zprávy

### PR DESCRIPTION
**Pull request by jkalend**
Adding #3

### CODE REVIEW

**Review by SafarRadim — COMMENTED** [review]
You left the, most likely unused, function "pin" in there

**Comment by SafarRadim** [comment]
And also the amount might benefit from being put in .env

File: frontroomsbot/bot.py [hunk 1]
```diff
@@ -84,6 +89,71 @@ async def on_raw_reaction_add(payload: discord.RawReactionActionEvent):
...
+        ):
+            # FIXME
+            # we need to maintain when was the last timeout,
+            # otherwise someone could get locked out
+            duration = datetime.timedelta(minutes=config["reactions"]["timeout_duration"])
+            await message.author.timeout(duration)
+            break
+
+
+@client.event
```
Comment (S1ro1) [inline frontroomsbot/bot.py:152]: Remove this and then lgtm-ish

**Review by S1ro1 — APPROVED** [review]
lgtm

**Comment by S1ro1** [comment]
Not deleting the branch because of the FIXMEs, reuse it for that afterwards

**Pull request merged**